src/components/subscriptions/data/service.js

```diff
@@ -1,31 +1,27 @@
 import faker from 'faker';
 import moment from 'moment';
+import qs from 'query-string';
 
-import { PAGE_SIZE, ACTIVE, ASSIGNED, DEACTIVATED } from '../constants';
+import { ACTIVATED, ASSIGNED, DEACTIVATED } from '../constants';
+
+import apiClient from '../../../data/apiClient';
+import { configuration } from '../../../config';
 
 export function createSampleUser(licenseStatus) {
   return {
     userId: faker.random.uuid(),
     emailAddress: faker.internet.email(),
-    pendingSince: moment(faker.date.past(10)),
+    lastRemindDate: moment(faker.date.past(10)),
     licenseStatus,
   };
 }
 
-let users = [
-  [...Array(6)].map(() => createSampleUser(ACTIVE)),
+const users = [
```

Comment:
I'm a bit confused why we still need `users` here if we're no longer creating fake users?

Reply:
I have removed the functions that are no longer being used, but some are still in use and will be removed in the upcoming PRs. Some of the functions still in use require `users` variable.